### DISCUSSION TITLE
Design

### DISCUSSION THREAD

**Discussion by kiou98** [lschaeffer313/domaine-nc-mobile#69] [General]
Hello, je m'invite sur le projet. Je fais de l'UX/UI Design.
Vous avez des screen-shot, des brouillons ou même une description qui décrit un peu votre vision ?
Je peux vous proposer quelques pistes, si ça vous dit ?

**Comment by adriens** [comment]
Hello Vincent, mais carrément !!! J'ai parlé de toi hiersoir à la team @HakumenNC  et @lschaeffer313 

👉 Il faudrait que tu nous fiiles le mail utilisé sur ton android, tonc omtpe google afin que tu puisses arriver sur la beta, qu'en dis-tu ?

**Comment by kiou98** [comment]
yes c'est celui-là : ***@***.***
Cordialement,

Vincent Lavergne
@tealforge.com
UX Designer
00.687.94.43.63
***@***.***

**Reply by adriens** [reply]
ben let'sgo @lschaeffer313 pour @kiou98 🚀

**Reply by adriens** [reply]
du coup @kiou98 , pour android c'est cette adresse non ?
![image](https://user-images.githubusercontent.com/5235127/204980805-6f17f2f0-68e8-44f3-b2b6-f9bc12539289.png)

**Comment by kiou98** [comment]
yes

Cordialement,

Vincent Lavergne
@tealforge.com
UX Designer
00.687.94.43.63
***@***.***

**Comment by adriens** [comment]
@kiou98 , tu es sur la beta : tu peux nous valider que tu arrives bien à utiliser l'app :pray: :grey_question: ... en nous droppant un screenhot (par exemple tealforge ? :smile_cat: )

**Comment by kiou98** [comment]
Comment j'y accède ? Vous avez un APK, ou un lien ?

Cordialement,

Vincent Lavergne
@tealforge.com
UX Designer
00.687.94.43.63
***@***.***

**Comment by adriens** [comment]
> Comment j'y accède ? Vous avez un APK, 

Pas d'APK, on relese sur les stores en direct avec la CI

> ou un lien ?

of course : 

Check les liens utiles : https://github.com/lschaeffer313/domaine-nc-mobile#-liens-utiles

**Comment by kiou98** [comment]
Merci Adrien,
Avec la vidéo j'ai, je crois, un peu mieux compris le concept.
Bon, à vrai dire, c'est très/trop technique, et j'y voit peu d'impact (ce
que je cherche avant tout), du coups je ne suis personnelement pas
intéréssé.
Je préfère me concentrer sur des plateformes très accessibles, à tout un
chacun, des trucs qui brillent quoi.

Bon courage pour la suite et encore Bravo de créer en Open. C'est assez
rare (chez nous zote) pour être souligné.
A plus tard.

Cordialement,

Vincent Lavergne
@tealforge.com
UX Designer
00.687.94.43.63
***@***.***

**Reply by adriens** [reply]
Yep, c'est pas frocément grand publique.
merci pour les échanges, pas de souci, et cool pour les feedbacks.
Un jour ptet on fera u truc ensembles 🤞
